pkgs/development/python-modules/log-symbols/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, colorama
+, fetchPypi
+, isPy27
+, pytestCheckHook
+, lib }:
+
+buildPythonPackage rec {
+  pname = "log-symbols";
+  version = "0.0.14";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    # the package is named log-symbols but PyPI files are named log_symbols
+    pname = "log_symbols";
```

Comment:
I think the pname should be with _ but in top level python it should be with -. @jonringer

Reply:
this is correct. Some packages were added before PEP 503 was ratified, as long as the attr in python-packages.nix adheres to PEP503, then it's good.

Reply:
also, this gets embedded into the url used to fetch the package, so it has to match the pypi listing.